src/swap/components/SwapSettingsSlippageDescription.tsx

```diff
@@ -0,0 +1,18 @@
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageDescriptionReact } from '../types';
+
+export function SwapSettingsSlippageDescription({
```

Comment:
As I read this, I am wondering what's the difference between Description and Label for SwapSettingsSlippage.

Reply:
Updating Label to Title